application/controllers/admin/SurveymenuController.php

```diff
@@ -186,8 +186,12 @@ public function massDelete()
             $aSurveyMenuIds = json_decode(Yii::app()->request->getPost('sItems', '')) ?? [];
             $success = [];
             foreach ($aSurveyMenuIds as $menuid) {
-                $model = $this->loadModel($menuid);
-                $success[$menuid] = $model->delete();
+                $model = Surveymenu::model()->findByPk((int)$menuid);
+                $success[$menuid] = false;
+                if ($model !== null) {
+                    $model->delete();
```

Comment:
Must use `$success[$menuid] = $model->delete();` delete can return false

Reply:
This function is overwritten in the model...It returns nothing in this specific case

Reply:
Arg … no https://github.com/LimeSurvey/LimeSurvey/blob/ca2209caa2f525697331684d9722727f973d0fe5/application/models/Surveymenu.php#L604 `return parent::delete();`

I open a dev issue …